src/app/project/project.component.html

```diff
@@ -41,6 +41,8 @@
   </div>
 </div>
 
+<button (click)="startTour()">Start Tour</button>
```

Comment:
This of course isn't the final implementation, it's just for a bit easier development for now ;) 
Same for the related .ts file ;)

Reply:
Sure :) Let's just keep remember to move it to help panel before we will merge this.